### PR TITLE
Annotate spans with GKE/GCE information.

### PR DESCRIPTION
**Pull request by vadimberezniker**
---

**Version bump**: None

### CODE REVIEW

File: server/util/tracing/tracing.go [hunk 1]
```diff
@@ -147,10 +150,20 @@ func Configure(configurator *config.Configurator, healthChecker interfaces.Healt
 		return bsp.Shutdown(ctx)
 	})
 
+	ctx, cancel := context.WithTimeout(context.Background(), resourceDetectionTimeout)
+	defer cancel()
+	res, err := resource.New(ctx,
+		resource.WithDetectors(&gcp.GKE{}, &gcp.GCE{}),
```
Comment (tylerwilliams) [inline server/util/tracing/tracing.go:156]: can you:

gke := &gcp.GKE{}
resource.WithDetectors(gke, gke)

or do they need to be different?

**Review by tylerwilliams — APPROVED** [review]
LGTM

This looks fine but could you add a link to the description or explain what it does? (mostly curious here -- what GKE/GCE info gets attached?)

**Review by vadimberezniker — COMMENTED** [review]
> LGTM
> 
> This looks fine but could you add a link to the description or explain what it does? (mostly curious here -- what GKE/GCE info gets attached?)

https://github.com/buildbuddy-io/buildbuddy/pull/677#pullrequestreview-701347979

TL;DR GCE detector gives you VM information such as name, zone, type (VM name is probably the only thing we may care about)
GKE gives you the basic GKE attrs like pod name, namespace, cluster, etc
